src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.search-form {
+  margin-top: 20px;
+  display: flex;
+  position: relative;
+}
+
+.search-form__icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 50%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  transform: translate(0, -50%);
+}
+
+.search-form__icon--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+}
+
+.search-form__search {
+  display: block;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  font-family: Avenir, sans-serif;
```

Comment:
assign default text styles on entire document and just override them where you need.